autodocodec-api-usage/test_resources/nix/day-type.nix

```diff
@@ -1 +1 @@
-types.unspecified
+types.str
```

Comment:
I just realised you put this on top of the `autodocodec-nix` branch.
I guess that means I really have to finish that library now :P
(This is good!)